frontend/catalyst/from_plxpr.py

```diff
@@ -174,17 +175,20 @@ def handle_qnode(
     )
 
 
+# The map below describes the parity between PL transforms and Catalyst passes.
+# PL transforms having a Catalyst pass counterpart will have a name as value,
+# otherwise their value will be None.
 transforms_to_passes = {
     qml.transforms.cancel_inverses: "remove-chained-self-inverse",
     qml.transforms.merge_rotations: "merge-rotations",
+    qml.transforms.unitary_to_rot: None,
 }
 
 
-# This is our registration factory for those PL transforms having a Catalyst
-# pass counterpart. The map above describes the parity between PL and Catalyst,
-# whereas the loop below iterates across that map and generates a custom handler
-# for each transform. In order to ensure early binding, we make the Catalyst
-# pass an argument whose default value is set by the loop.
+# This is our registration factory for PL transforms. The loop below iterates
+# across the map above and generates a custom handler for each transform.
+# In order to ensure early binding, we make the Catalyst pass an argument
+# whose default value is set by the loop.
 for pl_transform, pass_name in transforms_to_passes.items():
     # pylint: disable=unused-argument, too-many-arguments, cell-var-from-loop
     @WorkflowInterpreter.register_primitive(pl_transform._primitive)
```

Comment:
@mudit2812 @albi3ro It looks we currently need to register all known PennyLane transform primitives, is there a way to get that list programatically from PennyLane, or maybe change the structure to avoid this?